sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/MathFunctionsSuite.scala

```diff
@@ -95,6 +94,19 @@ class MathFunctionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     checkEvaluation(c(Literal(1.0), Literal.create(null, DoubleType)), null, create_row(null))
   }
 
+  test("conv") {
```

Comment:
Yes, that's what hive does. If you don't have strong preference here, then let's keep it the same as hive for the invalid digit to meet the compatibility tests in udf_conv.q, and I would change to return null for the empty string.